src/dotnet-grpc/Properties/CoreStrings.resx

```diff
@@ -207,4 +207,7 @@
   <data name="ErrorOutputMustBeFilePath" xml:space="preserve">
     <value>Output path `{0}` is invalid. The path cannot be a directory path and must be a file path.</value>
   </data>
+  <data name="LogWarningNoReferenceResolved" xml:space="preserve">
+    <value>No file reference was found for pattern `{0}`.</value>
```

Comment:
What about:

> No file(s) found matching file argument `{0}`.

I prefer this because pattern isn't used in any other messages. Also this message is more clear about what the pattern was used for.